my_enumerable.rb

```diff
@@ -0,0 +1,14 @@
+module Enumerable
+    def all?
+      list.all? { |num| num < 5 }
+    end
+  
+    def any?
+      list.any? { |num| num == 2 }
+    end
```

Comment:
- [ ] You need to work again on the all and any, they are too specific, you need to take into consideration other cases where they should return false for example:
- list_of_nums.all? {|e| e > 5} should return false, but currently it returns true
- list_of_nums.any? {|e| e == 5} should return false, but currently it returns true